managedtenants/utils/ocm.py

```diff
@@ -162,6 +165,36 @@ def disable_addon(self, addon_id):
             f"/api/clusters_mgmt/v1/addons/{addon_id}", json={"enabled": False}
         )
 
+    def shortcircuit_migrate(self, addon_id):
+        def addon_migration_complete():
+            migration = self.get_addon_migration(addon_id).json()
+            return migration.get("state") == "completed"
```

Comment:
```suggestion
        addon_migration_complete = self.get_addon_migration(addon_id).json().get("state") == "completed"
```
wouldn't this be cleaner.
I don't understand how this function specifically helps :)